src/exampleTypes/Date.js

```diff
@@ -57,30 +66,21 @@ let DateComponent = ({
     <div>
       <RadioList
         options={F.autoLabelOptions(['exact', 'rolling'])}
-        value={node.range !== 'exact' ? 'rolling' : 'exact'}
+        value={range}
         style={{ marginBottom: 10 }}
-        onChange={value => {
-          tree.mutate(
-            node.path,
-            value === 'exact'
-              ? { range: 'exact', from: null, to: null }
-              : { range: '', from: null, to: null }
-          )
```

Comment:
Don't you still need to mutate? otherwise, you're not modifying the node itself

Reply:
You only need to mutate when you select the actual **date range** not when you switch from `exact` to `rolling`. 
When you select `exact` and chance any of the `to` and `from` we **mutate** ... same with when you select `rolling` and then select the range from the `select` box .... so no you do not have to in the act of selecting the type of range. That was the main issue here.

Reply:
But if you go from a specific date range in exact to rolling, you'll still have a filtered date range even though it's no longer visible. Likewise if you have a rolling range and then switch to non rolling, it will appear like there's no filter value even though you're still filtering by some range.

Reply:
The UI issue where the values persisted between the `exact/rolling` switch should be now resolved. We do not need to update the underlining node when we switch the type. Only when we select an actual value for the `rolling` range or `to/from` for the exact one.